go.mod

```diff
@@ -11,27 +11,32 @@ require (
 	github.com/disiqueira/gotree v1.0.0
 	github.com/docker/docker v1.13.1
 	github.com/ethereum/go-ethereum v1.9.6
-	github.com/filecoin-project/go-leb128 v0.0.0-20190212224330-8d79a5489543
+	github.com/go-interpreter/wagon v0.6.1-0.20200227184901-6803234760a6
+	github.com/golang/protobuf v1.3.2 // indirect
 	github.com/golang/snappy v0.0.1
-	github.com/golangci/golangci-lint v1.23.1 // indirect
 	github.com/google/go-cmp v0.3.1 // indirect
 	github.com/gtank/merlin v0.1.1 // indirect
 	github.com/ipfs/go-datastore v0.3.1
 	github.com/kylelemons/godebug v1.1.0 // indirect
 	github.com/libp2p/go-libp2p v0.5.1
 	github.com/libp2p/go-libp2p-core v0.3.0
 	github.com/libp2p/go-libp2p-kad-dht v0.5.0
+	github.com/mattn/go-colorable v0.1.4 // indirect
 	github.com/mattn/go-isatty v0.0.11 // indirect
 	github.com/multiformats/go-multiaddr v0.2.0
 	github.com/naoina/go-stringutil v0.1.0 // indirect
 	github.com/naoina/toml v0.1.1
+	github.com/onsi/ginkgo v1.11.0 // indirect
+	github.com/onsi/gomega v1.8.1 // indirect
```

Comment:
why gomega/ginkgo keeps coming back, are we using this anywhere ?

Reply:
good question, I just did `go mod tidy` to clean stuff up, same with below, I can try removing it since we aren't using it explicity